app/views/landing_page/blocks/_govspeak.html.erb

```diff
@@ -1,7 +1,11 @@
-<% inverse = options[:inverse] || false %>
+<% 
+  inverse = options[:inverse] || false
+  margin_bottom = block.data["margin_bottom"] || 9
```

Comment:
We don't want to include margins in the YAML because then content authors could break the spacing all over the page. Instead, if this has to be set, use an option inside the hero block, to set margin on govspeak only inside that block. See the card block for an example.